idd/Energy+.idd.in

```diff
@@ -75858,6 +75858,7 @@ EvaporativeFluidCooler:SingleSpeed,
        \minimum> 0.0
        \ip-units gal/min
   A4 , \field Performance Input Method
+       \required-field  
```

Comment:
Per the source code this input field is a required bu it has been missing in the IDD. This change just labels it required-field per the design.